tox.ini

```diff
@@ -19,14 +20,14 @@ commands =
     flake8 --config=tox.ini {posargs:.}
 
 [testenv:type]
-skipsdist = true
-skip_install = true
 deps =
     mypy
     django-stubs
     djangorestframework-stubs
+extras =
```

Comment:
I see that `boto3-stubs[s3]` is in the `[dev]` extra. All the type stubs ought to be in the same place. Either:
* Put all stubs in the `[dev]` extra (or another new extra name, if you want), so they can be easily installed for the benefit of developers' IDEs for auto-complete. This Tox env can then install that extra.
* Put all stubs under `[testenv:type]`

Reply:
Since we're going to be enforcing type checking in CI, I think it's reasonable to include this as a `[dev]` extra so developers get auto-complete in their IDEs.